src/dagre/acyclic.js

```diff
@@ -26,13 +26,13 @@ function dfsFAS(g) {
   var visited = {};
 
   function dfs(v) {
-    if (_.has(visited, v)) {
+    if (visited.hasOwnProperty(v)) {
```

Comment:
I don't have time to give a full review of this now, but to fix all the ESLint errors, we'd need to replace this with:

```suggestion
    if (Object.prototype.hasOwnProperty.call(visited, v)) {
```

We could also disable the ESLint rule, but considering the security implications/bugs the original might create (see https://eslint.org/docs/latest/rules/no-prototype-builtins), it's better to replace them with `Object.prototype.hasOwnProperty.call`.

I'd recommend **NOT** using [`Object.hasOwn`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn), since it's only supported by newer browsers, and I'm sure if we do use it, this might break people using older iPhones/mac computers.

Reply:
ok, thanks for your quick feedback!

I used `Object.prototype.hasOwnProperty.call` and fixed the tslint error.